packages/bot-web-ui/src/assets/sass/load-modal.scss

```diff
@@ -0,0 +1,290 @@
+            cursor: pointer;
+        }
+        &-explain {
+            font-size: var(--text-size-xxs);
+            margin: 8px auto 0;
+            text-align: left;
+            opacity: 0;
+        }
+        &-explain--show {
+            opacity: 1;
+            width: fit-content;
+        }
+    }
+    &__explain {
+        &-list {
+            margin-top: 5px;
+            line-height: 1.2;
+        }
+    }
+    &__title {
+        font-size: var(--text-size-s);
+        font-weight: 700;
+        margin: 15px;
+    }
+    &__container {
+        padding: 25px;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        height: $modal_container_height;
+    }
+    &__content {
+        margin-bottom: 25px;
+        display: flex;
+        justify-content: space-between;
+        min-height: 500px;
+    }
+    &__files {
+        width: 30%;
+    }
+    &__list {
+        overflow: auto;
+        height: 100%;
+    }
+    &__item {
+        padding: 20px 15px;
+        display: flex;
+        justify-content: space-between;
+        border-bottom: solid 1px var(--border-normal);
+
+        &-text {
+            margin-left: 10px;
+            display: flex;
+            flex-direction: column;
+            justify-content: space-between;
+            width: 50%;
+        }
+        &-title {
+            font-size: var(--text-size-s);
+        }
+        &-time {
+            margin-top: 5px;
+        }
+        &-location {
+            display: flex;
+            width: 50%;
+
+            > * {
+                align-self: center;
+            }
+        }
+        &-saved {
+            margin-left: 10px;
+            font-size: var(--text-size-s);
+            line-height: 1.43;
+        }
+        &--selected {
+            background-color: var(--state-hover);
+        }
+        > * {
+            align-self: center;
+        }
+    }
+    &__item:hover {
+        background-color: var(--state-hover);
+        cursor: pointer;
+    }
+    &__preview {
+        width: 65%;
+
+        &-workspace {
+            padding: 15px;
+            position: relative;
+            border-radius: $BORDER_RADIUS;
+            border: solid 1px var(--border-normal);
+            height: 100%;
+            text-align: center;
+
+            .injectionDiv {
+                pointer-events: none;
+            }
+        }
+        &-controls {
+            padding: 7px 5px;
+            display: flex;
+            flex-direction: column;
+            position: absolute;
+            right: 16px;
+            bottom: 16px;
+            border-radius: 30px;
+            background-color: $color-grey-2;
+            z-index: 99;
+            box-shadow: 2px 2px 5px var(--shadow-menu);
+        }
+        &-icon {
+            margin: 5px;
+            cursor: pointer;
+        }
+    }
+    &__footer {
+        position: absolute;
+        bottom: 0;
+        right: 25px;
+    }
+}
+
+.local {
```

Comment:
Could you scope this under `.load`? It'll prevent collision with other elements that have the same class.